tests/test_integrations/test_crowdstrike.py

```
@@ -23,22 +77,145 @@ def test_crowdstrike_steampipe_task(
+        actions = service.get_all_rest_api_actions()
+        action_ran = False
+        for action in actions:
+            if action.name != "Query sensors by filter":
+                continue
+            try:
+                task = sample_restapi_task(integration, action.code, action.parameters_definition)
+                result = handle_task(task)
+                print(result.model_dump_json(indent=2))
+                test_result_format(result)
+                action_ran = True
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)